src/onboarding/Start.js

```diff
@@ -205,7 +205,8 @@ class StartContent extends React.PureComponent {
       return (
         <ActionInfo>
           Please ensure your account has at least {MINIMUM_BALANCE / 10e18} ETH
-          (you have {Math.round(balance.toFixed() / 10e18 * 10e2) / 10e2} ETH).
+          (you have {Math.round((balance || 0).toFixed() / 10e18 * 10e2) / 10e2}{' '}
```

Comment:
I've added a few rounding utils that are _potentially_ more accurate, see https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/src/lib/math-utils.js#L12